exec/java-exec/src/test/java/org/apache/drill/exec/store/TestFormatPluginLoading.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.drill.exec.store;
+
+import org.apache.drill.common.logical.StoragePluginConfig;
+import org.apache.drill.exec.ExecConstants;
+import org.apache.drill.exec.ExecTest;
+import org.apache.drill.exec.store.dfs.FileSystemConfig;
+import org.apache.drill.exec.store.easy.text.TextFormatPlugin;
+import org.apache.drill.exec.store.sys.PersistentStore;
+import org.apache.drill.shaded.guava.com.google.common.io.Resources;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestFormatPluginLoading extends ExecTest {
+  private final StoragePluginRegistryImpl storagePluginRegistry;
+  private final PersistentStore<StoragePluginConfig> plugins;
+
+  private static File storagePlugins;
+  private static File formatPlugins;
+
+  public TestFormatPluginLoading() throws Exception {
+    storagePluginRegistry = new StoragePluginRegistryImpl(mockDrillbitContext());
+    storagePluginRegistry.init();
+    plugins = storagePluginRegistry.getStore();
+  }
+
+  @BeforeClass
+  public static void setup() {
+    storagePlugins = new File(Resources.getResource("store/" + ExecConstants.BOOTSTRAP_STORAGE_PLUGINS_FILE).getFile());
```

Comment:
Can you please explain what you are doing here? I am sure what this renaming logic does.